metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -541,6 +546,9 @@ def _relationships_endpoint(self):
     def _aspect_count_endpoint(self):
         return f"{self.config.server}/aspects?action=getCount"
 
+    def _session(self) -> Session:
+        return super()._session
```

Comment:
this doesn't look right

I don't think it's necessary, unless you want _session to be a method instead of an attr

Reply:
I was not able in the DataHubGraph object to pull attributes from the super class, so I built this function to surface that. At least the auto-import in Intellij wasn't finding it.

What would be the right way to surface this `super`'s attribute?

Reply:
you should be able to do graph._session without any changes

if you really want a method, add a `@property` decorator